services/kommander/0.2.0/defaults/cm.yaml

```diff
@@ -88,7 +88,7 @@ data:
       prerequisites:
         defaultApps:
           reloader: "0.0.104"
-          traefik: "10.9.1"
+          traefik: "10.9.2"
```

Comment:
[nit] it would be great if we create another PR for rest of changes since they seems not related with renaming

Reply:
I'm not sure I understand what you'd like me to do.

Reply:
I'm not changing the version of traefik itself, only the App version so as to make it upgradeable.

Reply:
Ohh, it makes sense. Confused because of the version schema.

Reply:
Just realized this change is going to break e2e tests. In some parts of apps, we're still assuming the app version equal to the chart version. 

However, we need to change those assumptions anyway